makefile

```diff
@@ -19,4 +19,9 @@ desktop:
 	cd build; LIBRARY_PATH=../install_root/lib make 
 	cd build; LIBRARY_PATH=../install_root/lib make install
 
-all: desktop mobile
+check:
+	if test ! -d build; then (echo "error: please run build.sh before make"; exit -1;); fi
+	cd build; LIBRARY_PATH=../install_root/lib make check
```

Comment:
does this work? That looks like it would use `subsurface/install_root/lib` as LIBRARY_PATH and that seems wrong... because 
a) it should be subsurface/../install/root/lib
b) it should be DYLD_LIBRARY_PATH=../../install-root/lib 
c) even with that, for me calling the individual tests works, but calling make check fails (and I can't figure out why)